packages/features/bookings/components/AvailableTimes.tsx

```diff
@@ -97,7 +99,10 @@ export const AvailableTimes = ({
               data-disabled={bookingFull}
               data-time={slot.time}
               onClick={() => onTimeSelect(slot.time, slot?.attendees || 0, seatsPerTimeSlot, slot.bookingUid)}
-              className="min-h-9 hover:border-brand-default mb-2 flex h-auto w-full flex-col justify-center py-2"
+              className={classNames(
+                "min-h-9 hover:border-brand-default mb-2 flex h-auto w-full flex-col justify-center py-2",
+                selectedSlots?.includes(slot.time) && "border-brand-default"
+              )}
```

Comment:
Here we're just checking from the provided selectedSlots and checking the active status of the button from that